ocm_python_wrapper/cluster.py

```diff
@@ -388,3 +396,24 @@ def _wait_for_rhmi_resource():
         ResourceEditor(
             patches={rhmi: {"spec": {"useClusterStorage": "false"}}}
         ).update()
+
+    @staticmethod
+    def create_rhods_brew_config():
+        if not os.getenv("PS_DATA"):
+            raise KeyError("Pull-secret data must be specified")
+
+        brew_icsp = create_icsp(
+            icsp_name="brew-registry",
+            repository_digest_mirrors=[
+                {
+                    "source": "registry.redhat.io/rhods",
+                    "mirrors": ["brew.registry.redhat.io/rhods"],
+                }
+            ],
+        )
+        brew_pull_secret = create_update_pull_secret(
+            new_content_dict=json.loads(os.environ["PS_DATA"]),
```

Comment:
`os.environ["PS_DATA"]` save and use it.

Reply:
done